docs/_app.js

```diff
@@ -46,38 +53,35 @@ const pageNames = {
   'sync-api': 'Sync API'
 }
 
-const sortRoutes = routes => [
-  ...sortBy([...routes], a => {
-    const i = navOrder.indexOf(a.name)
-    return i < 0 ? Infinity : i
+const sortRoutes = routes =>
+  [
+    ...sortBy([...routes], a => {
+      const i = navOrder.indexOf(a.name)
+      return i < 0 ? Infinity : i
+    })
+  ].map(route => {
+    if (!pageNames[route.name]) return route
+    return {
+      ...route,
+      name: pageNames[route.name]
+    }
   })
-].map(route => {
-  if (!pageNames[route.name]) return route
-  return {
-    ...route,
-    name: pageNames[route.name]
-  }
-})
 
 export default class App extends React.Component {
-  static defaultProps = {
-    title: 'MDX'
-  }
-
-  render () {
+  render() {
     const { routes } = this.props
     const nav = sortRoutes(routes)
 
     return (
       <RebassMDX>
         <ScopeProvider scope={scope}>
-          <SidebarLayout
-            {...this.props}
-            logo={<Logo />}
-            routes={nav}
-          />
+          <SidebarLayout {...this.props} logo={<Logo />} routes={nav} />
         </ScopeProvider>
       </RebassMDX>
     )
   }
 }
+
+App.defaultProps = {
+  title: 'MDX'
+}
```

Comment:
ESLint doesn't support class properties yet and I didn't want to introduce babel-eslint just for this.

Reply:
I didn't touch any JavaScript code except this. Everything else is Prettier.